native/cocos/engine/Engine.cpp

```diff
@@ -51,6 +51,8 @@
 #include "cocos/network/HttpClient.h"
 #include "core/Root.h"
 #include "core/assets/FreeTypeFont.h"
+#include "editor-support/dragonbones-creator-support/ArmatureCacheMgr.h"
+#include "editor-support/spine-creator-support/SkeletonCacheMgr.h"
```

Comment:
spine and dragonbones may not be included into engine. Need to use corresponding macro to include these codes.

Reply:
Where is the corresponding macro definition

Reply:
You can find them in CMakeLists.txt, such as CC_USE_SPINE.